src/igvfd/searches/configs/prediction_set.py

```diff
@@ -9,102 +9,188 @@ def prediction_set():
+                'category': 'Prediction Set Details',
+                'description': 'The scope or scale that this prediction set is designed to target.'
             },
             'software_versions.software.title': {
                 'title': 'Software',
+                'category': 'Prediction Set Details',
+                'description': 'The software used to produce this prediction.'
             },
             'associated_phenotypes.term_name': {
-                'title': 'Associated Phenotypes'
+                'title': 'Associated Phenotypes',
+                'category': 'Prediction Set Details',
+                'description': 'Ontological terms for diseases or phenotypes associated with this prediction set.'
             },
             'assessed_genes.symbol': {
                 'title': 'Assessed Genes',
+                'optional': True,
+                'category': 'Prediction Set Details',
+                'description': 'A list of gene(s) assessed in this prediction set, especially those investigated for functional impacts.'
             },
             'donors.taxa': {
                 'title': 'Taxa',
+                'optional': True,
+                'category': 'Sample',
+                'description': 'The organism or species associated with the donor.'
             },
             'samples.classifications': {
                 'title': 'Classification',
+                'optional': True,
+                'category': 'Sample',
+                'description': 'High-level classification of the sample.'
             },
             'samples.sample_terms.term_name': {
                 'title': 'Sample',
+                'category': 'Sample',
+                'description': 'The sample term name associated with the prediction set.'
             },
             'samples.targeted_sample_term.term_name': {
                 'title': 'Cellular Transformation Target',
+                'optional': True,
+                'category': 'Sample',
+                'description': 'Ontology term identifying the targeted endpoint sample resulting from differentiation or reprogramming.'
             },
             'samples.disease_terms.term_name': {
                 'title': 'Disease',
+                'optional': True,
+                'category': 'Sample',
+                'description': 'Ontology term of the disease associated with the sample.'
             },
             'samples.growth_medium': {
                 'title': 'Growth Medium',
+                'optional': True,
+                'category': 'Sample',
+                'description': 'The growth medium or conditions used to culture the sample.'
             },
             'samples.modifications.modality': {
-                'title': 'Modification'
+                'title': 'Modification',
+                'optional': True,
+                'category': 'Sample',
+                'description': 'The type of modification applied to the sample.'
```

Comment:
"The intended effect of an exogenous or in situ modification of the sample; e.g. activation, inhibition, degradation"